ggml/src/ggml-cuda/common.cuh

```diff
@@ -211,6 +209,10 @@ typedef float2 dfloat2;
 #define FP16_MMA_AVAILABLE
 #endif // defined(GGML_HIP_ROCWMMA_FATTN) && (defined(CDNA) || defined(RDNA3) || defined(RDNA4))
 
+#if defined(GGML_USE_MUSA)
+#define FP16_MMA_AVAILABLE
+#endif // defined(GGML_USE_MUSA)
```

Comment:
Add this to the check for CUDA instead, the correct logic should be:

```C
#if GGML_USE_MUSA || (!defined(GGML_USE_HIP) && __CUDA_ARCH__ >= GGML_CUDA_CC_VOLTA)
```

(In a discussion somewhere else we determined that the check for `__HIP_PLATFORM_AMD__` is not actually needed and just makes the code more cumbersome.)